moveit_core/robot_trajectory/src/robot_trajectory.cpp

```diff
@@ -214,12 +214,17 @@ RobotTrajectory& RobotTrajectory::unwind(const moveit::core::RobotState& state)
     double running_offset = reference_value0 - reference_value;
 
     double last_value = waypoints_[0]->getJointPositions(cont_joint)[0];
```

Comment:
I'm confused why this is named `last_value`? It looks like `waypoint_value` would be a better name.